src/sentry/static/sentry/app/views/settings/organizationDeveloperSettings/sentryApplicationRow.jsx

```diff
@@ -0,0 +1,182 @@
+import React from 'react';
+import {Box, Flex} from 'grid-emotion';
+import {Link} from 'react-router';
+import Button from 'app/components/button';
+import Confirm from 'app/components/confirm';
+import SentryAppAvatar from 'app/components/avatar/sentryAppAvatar';
+import PropTypes from 'prop-types';
+import SentryTypes from 'app/sentryTypes';
+import {PanelItem} from 'app/components/panels';
+import {t} from 'app/locale';
+import styled from 'react-emotion';
+import space from 'app/styles/space';
+import {withTheme} from 'emotion-theming';
+import CircleIndicator from 'app/components/circleIndicator';
+
+export default class SentryApplicationRow extends React.PureComponent {
+  static propTypes = {
+    app: SentryTypes.SentryApplication,
+    orgId: PropTypes.string.isRequired,
+    installs: PropTypes.array,
+    onInstall: PropTypes.func,
+    onUninstall: PropTypes.func,
```

Comment:
The two above here should probably be `isRequired` as well, since we're just optimistically calling them down below.